### ISSUE TITLE
Support for LinkedIn

### ISSUE THREAD

**Issue by allanbowe** [pseudozach/nostr.directory#1]
Given the recent censorship of Nostr on Twitter:  https://twitter.com/TwitterSupport/status/1604531261791522817

Could this website be pivoted towards other services, such as Github and LinkedIn?

**Comment by pseudozach** [comment]
yes, there's definitely room to validate other social media. we could do an ad-hoc check for a public gist on github or a linked in post.
or maybe they just publish their pubkey and verify on nostr like twitter. so we just modify the verification listener to parse social media type.
